sdk/src/main/java/me/digi/sdk/interapp/managers/DMENativeConsentManager.kt

```diff
@@ -97,6 +98,15 @@ class DMENativeConsentManager(val sessionManager: DMESessionManager, val appId:
 
     override fun handle(intent: Intent?) {
 
+        val ctx = DMEAppCommunicator.getSharedInstance().context
+        val params = intent?.extras?.toMap() ?: emptyMap()
+
+        val (errorCode, errorMessage, errorReference) = Triple(
```

Comment:
This is nice, I didn't know you could assign the result of a triple in this way, so 👍 for teaching me something new. 😄

Reply:
You can either pass variable names in the parenthesis, or you can have a variable with a single name and then access components within the Triple with either variable_name.first, variable_name.second, or variable_name.component1(), variable_name.component2(). However I prefer this way because it's much more readable since I'm able to access variable directly by its name as you saw here :D